### ISSUE TITLE
resource fortiswitch_switch_interface not working

### ISSUE THREAD

**Issue by thekerneldump** [fortinetdev/terraform-provider-fortiswitch#1]
This sample block:

`resource "fortiswitch_switch_interface" "port2" {
  name = "port2"
  native_vlan = "106"

}`

returns a 500 from Fortiswitch 7.0.1.  When doing a debug on the switch for the https interface, the following is happening:

`api_cmdb.c    [ 2128] : editing CMDB object : append (1) auto_key (0) mkey () flags (0) ...
api_json.c    [ 1975] : setting attribute 'name' : 'interface'
api_json.c    [ 1975] : setting attribute 'native-vlan' : '106'Error in cmf_shm_api.c,cmf_shm_update,765, pid=867,vd=root,query=switch.interface,level=1,pos=0.
0: config switch interface
1: edit "interface"
(skipped) set native-vlan 106`

It appears the port is not being sent as "port2" (which is defined in the TF code), but rather the name of the interface sent is "interface".  which is invalid.

**Comment by MaxxLiu22** [comment]
Hi @thekerneldump ,

Thank you for raising this question, if we would like to use terraform to manage an existing object, we have to import first by using ``terraform import fortiswitch_switch_interface.port2 port2``, but there exists a data type issue that has been reported to the development team, now we can use generic method as a temporary solution. I will let you know once this issue has been fixed.
```
resource "fortiswitch_json_generic_api" "trname" {
path = "/api/v2/cmdb/switch/interface/port2"
method = "PUT"
  json   = <<EOF
{
  "native-vlan" : "108"
}
EOF
 }
```

Thanks,
Maxx

**Comment by MaxxLiu22** [comment]
Hi @thekerneldump ,

The issue has been fixed in Terraform FSW 1.1.2. Please upgrade to this version and give it a try. I will proceed to close this case. Feel free to reopen it if the issue persists.

Thanks,
Maxx
